dist/main.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * ATTENTION: The "eval" devtool has been used (maybe by default in mode: "development").
+ * This devtool is neither made for production nor for readable output files.
+ * It uses "eval()" calls to create a separate source file in the browser devtools.
+ * If you are trying to read the output file, select a different devtool (https://webpack.js.org/configuration/devtool/)
+ * or disable the default devtool with "devtool: false".
+ * If you are looking for production-ready output files, see mode: "production" (https://webpack.js.org/configuration/mode/).
+ */
+/******/ (() => { // webpackBootstrap
+/******/ 	"use strict";
+/******/ 	var __webpack_modules__ = ({
```

Comment:
- [ ] Kindly configure your webpack correctly, you're missing important files such as `package.json`, `webpack.config` e.t.c. Watch this short [tutorial](https://www.youtube.com/watch?v=HNb6bapmsyI)
- [ ] the following scripts must be present in your `package.json` for running and bundling the app
![image](https://user-images.githubusercontent.com/81932478/208930889-e8b1100d-f165-453b-9fe9-a9ddd95def7e.png)